### PR TITLE
Use bundleId and windowId on Mac; fix docs (closes #72)

### LINKED ISSUE #72: Choose Application dialog appears on find-window execution when Electron app is running

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @georgiy-abbasov @AlexanderMoskovkin 

Tested on Yosemite VM and the real Mac.

@VasilyStrelyaev, please, check changes in the docs.

Changed scripts:
```applescript
# close.scpt

on run (argv)
	set windowId to (item 1 of argv)
	
	set bundleId to (item 2 of argv)
	
	tell application id (bundleId as string) to close window id windowId
end run
```
```applescript
# find-window.scpt

on run (argv)
	if (count of argv) is not equal to 1 then
		error "Incorrect arguments"
	end if
	set windowNamePart to item 1 of argv
	
	repeat 10 times
		try
			set {bundleId, windowId} to findAppByWindowTitle(windowNamePart)
			
			if bundleId is not equal to "" then
				return bundleId & linefeed & windowId
			end if
		end try
		
		#300ms delay
		delay 0.3
	end repeat
	
	error "Window not found"
end run

on findAppByWindowTitle(windowNamePart)
	tell application "System Events"
		set bundleIds to (get bundle identifier of every process whose background only is false)
	end tell
	
	repeat with bundleId in bundleIds
		
		tell application id bundleId
			set windowsNamesList to {}
			try
				set windowsNamesList to (get name of every window)
			end try
			repeat with windowName in windowsNamesList
				if (windowName as string) contains windowNamePart then
					set windowId to id of window windowName
					return {(bundleId as string), (windowId as string)}
				end if
			end repeat
		end tell
		
	end repeat
	
	return ""
end findAppByWindowTitle
```
```applescript
# get-window-size.scpt:

on run (argv)
	if (count of argv) is not equal to 4 then
		error "Incorrect arguments"
	end if
	
	set windowId to item 1 of argv
	set bundleId to item 2 of argv
	set newWidth to item 3 of argv
	set newHeight to item 4 of argv
	
	tell application id (bundleId as string) to set oldWindowBounds to bounds of window id windowId
	
	set {newX, newY} to oldWindowBounds
	
	tell application id (bundleId as string)
		set bounds of window id windowId to {newX, newY, newX + newWidth, newY + newHeight}
	end tell
end run
```
```applescript
# resize.scpt:

on run (argv)
	if (count of argv) is not equal to 4 then
		error "Incorrect arguments"
	end if
	
	set windowId to (item 1 of argv as integer)
	set bundleId to (item 2 of argv as string)
	set newWidth to (item 3 of argv as integer)
	set newHeight to (item 4 of argv as integer)
	
	tell application id bundleId to set oldWindowBounds to bounds of window id windowId
	
	set {newX, newY} to oldWindowBounds
	
	tell application id bundleId
		set bounds of window id windowId to {newX, newY, newX + newWidth, newY + newHeight}
	end tell
end run
```

### CODE REVIEW

**Comment by inikulin** [comment]
lgtm

File: test/playground/routes.js [hunk 1]
```diff
@@ -66,19 +61,14 @@ function objectToList (object, keyName) {
...
-        getDeviceNames()
-            .then(function (res) {
-                deviceNames = res;
-            })
-    ]);
+    port        = appPort;
+    deviceNames = getDeviceNames();
+
+    return browserNatives.getInstallations()
+        .then(function (res) {
```
Comment (georgiy-abbasov) [inline test/playground/routes.js]: Put the newline before `getInstallations()` please.
```js
return browserNatives
         .getInstallations()
         .then(function (res) {
```

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by georgiy-abbasov** [comment]
lgtm

File: README.md [hunk 2]
```diff
@@ -23,11 +23,9 @@ $ npm install testcafe-browser-natives
...
 #API Reference
 
-**Important note**: Most of the provided functions use a page URL to identify a browser window. However, depending on a browser, a browser window can be identified by a page URL or a web page title. Therefore, before calling the functions, you need to pass the page URL to the document title. Here is an example:
-```js
-document.title = document.location.toString()
-```
-Also, note that Microsoft Edge truncates the window title to the hostname if document.title contains only a URL
+**Important note**: Most of the provided functions use a title of the web browser window to identify it.
+The window title equals the web page's document.title for most of the browsers.
+However, Microsoft Edge truncates the window title to the hostname if document.title contains only a URL
```
Comment (VasilyStrelyaev) [inline README.md]:
```
**Important note**: To identify the web browser window, most of the provided functions use its title. This title matches the web page's document.title in most of the browsers. However,
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by AndreyBelym** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**